frontend/src/LandingView/styles/LogoBannerStyled.js

```diff
@@ -18,6 +18,10 @@ const LogoBannerStyles = styled.div`
 	);
 `;
 
+const TextIMG = styled.img`
+	height: 80vh;
```

Comment:
Unless you want something to scale the entirety of the width or height of your view port, you should avoid using vw and vh due to inconsistent sizing across browsers.